omod-1.9/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_9/ProviderAttributeTypeResource1_9.java

```diff
@@ -9,12 +9,14 @@
  */
 package org.openmrs.module.webservices.rest.web.v1_0.resource.openmrs1_9;
 
+import io.swagger.models.Model;
```

Comment:
Is this import being used here? Get rid of all unused imports in all the files if they exist